src/components/static/Home.tsx

```diff
@@ -71,13 +71,13 @@ export const Home: React.FunctionComponent = () => {
               </Typography>
             </div>
             <Grid container justify="center" alignItems="center">
-              <NavLink to={ROUTES.ELIGIBILITY} className="home__nav-link">
+              <NavLink to={ROUTES.ABOUT} className="home__nav-link">
```

Comment:
When redirect, the about page should start at top , it renders in the middle for now